packages/pn-personafisica-webapp/src/components/Contacts/CourtesyContactItem.tsx

```diff
@@ -14,34 +14,30 @@ import DeleteDialog from './DeleteDialog';
 import DigitalContactElem from './DigitalContactElem';
 import { useDigitalContactsCodeVerificationContext } from './DigitalContactsCodeVerification.context';
 
-export enum CourtesyFieldType {
-  EMAIL = 'email',
-  PHONE = 'phone',
-}
-
 interface Props {
-  type: CourtesyFieldType;
+  type: CourtesyChannelType.EMAIL | CourtesyChannelType.SMS;
   value: string;
   blockDelete?: boolean;
 }
 
 const CourtesyContactItem = ({ type, value, blockDelete }: Props) => {
   const { t } = useTranslation(['common', 'recapiti']);
   const { initValidation } = useDigitalContactsCodeVerificationContext();
-  const [phoneRegex, setPhoneRegex] = useState(dataRegex.phoneNumber);
+  const phoneRegex = value ? dataRegex.phoneNumberWithItalyPrefix : dataRegex.phoneNumber;
   const digitalElemRef = useRef<{ editContact: () => void }>({ editContact: () => {} });
   const [showDeleteModal, setShowDeleteModal] = useState(false);
   const dispatch = useAppDispatch();
 
   const digitalDomicileType = useMemo(
-    () => (type === CourtesyFieldType.EMAIL ? CourtesyChannelType.EMAIL : CourtesyChannelType.SMS),
+    () =>
+      type === CourtesyChannelType.EMAIL ? CourtesyChannelType.EMAIL : CourtesyChannelType.SMS,
     []
   );
 
   const emailValidationSchema = useMemo(
     () =>
       yup.object().shape({
-        email: yup
+        EMAIL: yup
```

Comment:
define a variable const contactType = type.toLowerCase() and use it around the code instead of type.toLowerCase()